examples/difference_in_mapcubes.py

```diff
@@ -0,0 +1,42 @@
+"""
+========================================
+Running Difference in Mapcubes
+========================================
+
+This example illustrates how to make running differencing within a MapCube.
+"""
+
+import matplotlib.pyplot as plt
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+
+############################################################################
+# We first loop through all the maps in the aiamapcube and differentially rotate each map
+# in the mapcube with respect to the previous map.
+# We then subtract each map data from the previous map and store all such difference maps in a list.
+diffmap = []
+for i in range(0, len(aiamapcube) - 1):
+    # take the difference
+    aiamapcube_rot = diffrot.diffrot_map(aiamapcube[i+1], time=aiamapcube[i].date)
+    diffdata = aiamapcube[i+1].data - aiamapcube_rot.data
```

Comment:
This is wrong, you are subtracting the same image but rotated. I think what you want to do here is:
```python
diffdata = aiamapcube_rot.data - aiamapcube[i].data
```

Reply:
Ahh!, I didn't notice that.